flint.ui/src/components/Cards/CardInfoConfig.vue

```diff
@@ -11,36 +11,42 @@
           <span class="font-semibold text-xl text-blueGray-700">
             {{ cardTitle }}
           </span>
-          <button @click="showConfirmRunModal()" class="w-full mt-4 block align-middle flex-initial bg-white hover:bg-black hover:text-white text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded shadow">
-           <i class="far fa-play-circle"></i> {{cardFunctionRun}}
+          <button
+            @click="showConfirmRunModal()"
+            class="w-full mt-4 block align-middle flex-initial bg-white hover:bg-black hover:text-white text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded shadow"
```

Comment:
General question, not specifically for this PR. Can we define style classes elsewhere to make the template code easier to read?